tmt/steps/provision/__init__.py

```diff
@@ -625,6 +625,12 @@ def is_ready(self) -> bool:
 
         raise NotImplementedError
 
+    @property
+    def log_names(self) -> list[str]:
+        """Return name list of logs the guest could provide."""
```

Comment:
Check out other one-line docstrings - if they fit on one line, there should be a space before and after the text, and no full stop at the end of sentence.